pubsub.go

```diff
@@ -22,6 +22,14 @@ import (
 
 var log = logging.Logger("pubsub-valuestore")
 
+type watchGroup struct {
+	lk      sync.RWMutex
+	closing chan struct{}
+
+	listeners map[int64]chan<- []byte
```

Comment:
We can probably just make this a `map[chan<- []byte]struct{}`. Then we don't need to give each listener an ID.